src/styles/cylc/_tree.scss

```diff
@@ -0,0 +1,35 @@
+@import '~vue-virtual-scroller/dist/vue-virtual-scroller.css';
+@import '~infinite-tree/dist/infinite-tree.css';
+@import '~vuetify/src/styles/styles.sass';
+
+$active-color: #BDD5F7;
+$arrow-size: 15px;
+$leaf-background-color: map-get($grey, 'lighten-3');
+
+.c-tree {
+  .node {
+    line-height: 1.8em;
+
+    .node-data {
+      margin-left: 12px;
```

Comment:
Should we use `em` for `margin-left` (and also for `padding-left` in the infinite tree code)?

Reply:
Oh, will have to look with more calm why I used `12px`. :+1:

Reply:
It's not a blocker or anything just a quick comment, because the `line-height` is set in `em` above.